src/lib/utils/index.ts

```diff
@@ -73,3 +73,19 @@ export const RUST_ISO_UTC_DATE_FORMAT = 'yyyy-MM-dd HH:mm:ss';
 export function isTouchScreenOnly(): boolean {
   return window.matchMedia('(hover: none) and (pointer: coarse)').matches;
 }
+
+export function getCookie(cname: string) {
+  let name = cname + '=';
+  let decodedCookie = decodeURIComponent(document.cookie);
+  let ca = decodedCookie.split(';');
+  for (let i = 0; i < ca.length; i++) {
+    let c = ca[i];
+    while (c.charAt(0) == ' ') {
+      c = c.substring(1);
+    }
+    if (c.indexOf(name) == 0) {
+      return c.substring(name.length, c.length);
+    }
```

Comment:
could this be simplified to
```suggestion
    let c = ca[i].trim();
    if (c.startsWith(name)) {
      return c.substring(name.length, c.length);
    }
```

I think you could further simplify the function by not appending `=` to the name and then just doing

```
return c.substring(name.length + 1, c.length);
```

Reply:
good suggestion gonna simplify